src/components/security_manager/test/security_manager_test.cc

```diff
@@ -87,6 +89,8 @@ uint8_t handshake_data_out[] = {0x6, 0x7, 0x8};
 uint8_t* handshake_data_out_pointer = handshake_data_out;
 const size_t handshake_data_out_size =
     sizeof(handshake_data_out) / sizeof(handshake_data_out[0]);
+const uint32_t kAsyncExpectationsTimeout = 10000u;
```

Comment:
@istoilovgithub Usually it is good when we have time-out variables or consts to put the unit in the name like uS, Ms, Sec etc. So, how much time is this 10000? (check all)